src/plugins/kibana_react/common/eui_provider.tsx

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
This file should probably be local to `kbn-storybook` and exported from there... no reason to introduce it to `kibana_react` when the only consumer is `kbn-storybook` and its dependents, right?

Reply:
Yep, good call. Will update.